File: core/identity/timezone_runtime_engine.py
from __future__ import annotations

from typing import Any, Dict

_TIMEZONES = {
    "default": "America/New_York",
    "profile_a": "Europe/London",
    "profile_b": "America/Los_Angeles",
}


def build_timezone_runtime(profile_id: str = "default") -> Dict[str, Any]:
    profile = profile_id if profile_id in _TIMEZONES else "default"

    return {
        "timezone": _TIMEZONES[profile],
        "bounded": True,
    }

File: src/identity/timezoneRuntimeEngine.ts
/**
 * Converted from Python: core/identity/timezone_runtime_engine.py
 * @generated — WebWeaveX python→javascript library port
 */

import * as py from "../runtime/pyCompat.js";

let _TIMEZONES: any = {"default": "America/New_York", "profile_a": "Europe/London", "profile_b": "America/Los_Angeles"};
export function buildTimezoneRuntime(profile_id: any = "default"): any {
  var profile: any = (py.contains(_TIMEZONES, profile_id) ? profile_id : "default");
  return {"timezone": py.at(_TIMEZONES, profile), "bounded": true};
}
